pallets/amm/README.md

```diff
@@ -0,0 +1,16 @@
+### AMM (Automated Market Maker) pallet
+
+Intro: https://gist.github.com/unordered-set/50d601ff567094107d3bb1061dad5234
```

Comment:
I'd remove both of these. This is not really official docs grade (even though it's quite good). The presentation is about exchange not amm.